Sources/AppStoreServerLibrary/ChainVerifier.swift

```diff
@@ -120,7 +119,7 @@ class ChainVerifier {
         return verificationResult
     }
     
-    func verifyChainWithoutCaching(leaf: Certificate, intermediate: Certificate, online: Bool, validationTime: Date) async -> X509.VerificationResult {
+    nonisolated func verifyChainWithoutCaching(leaf: Certificate, intermediate: Certificate, online: Bool, validationTime: Date) async -> X509.VerificationResult {
         var verifier = Verifier(rootCertificates: self.store) {
```

Comment:
Since `Verifier` is _not_ `Sendable`, but becomes tied to the actor, we must mark this method `nonisolated` to detach this dependency and allow it to be safely used within the context of this method.